### ISSUE TITLE
Add Custom gradient operators for Jax Frontend

### ISSUE THREAD

**Issue by Aarsh2001** [unifyai/ivy#1580]
Add [Custom gradient operators](https://jax.readthedocs.io/en/latest/jax.lax.html#control-flow-operators) for Jax frontend:

- [ ] stop_gradient
- [ ] custom_linear_solve
- [ ] custom_root

**Comment by cecethea** [comment]
- [ ] #2501

**Comment by ukj02** [comment]
custom_linear_solve

**Comment by ukj02** [comment]
#3431
